tensorflow/core/kernels/dynamic_partition_op_gpu.cu.cc

```diff
@@ -348,10 +352,13 @@ class DynamicPartitionOpGPU : public AsyncOpKernel {
             &cub_temp_storage),
         done);
     // Radix-sort the partition information.
-    gpuprim::DeviceRadixSort::SortPairs(
-        cub_temp_storage.flat<int8>().data(), temp_storage_bytes,
-        partitions_ptr, partitions_out_ptr, indices_in_ptr, indices_out_ptr, N,
-        0, sizeof(int32) * 8, cu_stream);
+    gpuResult = gpuprim::DeviceRadixSort::SortPairs(
+              cub_temp_storage.flat<int8>().data(), temp_storage_bytes,
+              partitions_ptr, partitions_out_ptr, indices_in_ptr, indices_out_ptr, N,
+              0, sizeof(int32) * 8, cu_stream);
+    
+    OP_REQUIRES(c, gpuResult == gpuSuccess, 
+            errors::Internal("Failed to launch copy from device to host."));
```

Comment:
errors::Internal("Failed to launch gpuprim::DeviceRadixSort::SortPairs, temp_storage_bytes: " temp_storage_bytes, ", status: ", cudaGetErrorString(gpuResult));